packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/InAppPurchasePluginTests.m

```diff
@@ -299,89 +285,55 @@ - (void)testRestoreTransactions {
 }
 
 - (void)testRetrieveReceiptDataSuccess {
-  XCTestExpectation *expectation = [self expectationWithDescription:@"receipt data retrieved"];
-  FlutterMethodCall *call = [FlutterMethodCall
-      methodCallWithMethodName:@"-[InAppPurchasePlugin retrieveReceiptData:result:]"
-                     arguments:nil];
-  __block NSDictionary *result;
-  [self.plugin handleMethodCall:call
-                         result:^(id r) {
-                           result = r;
-                           [expectation fulfill];
-                         }];
-  [self waitForExpectations:@[ expectation ] timeout:5];
+  FlutterError *error;
+  id result = [self.plugin retrieveReceiptDataWithError:&error];
   XCTAssertNotNil(result);
   XCTAssert([result isKindOfClass:[NSString class]]);
```

Comment:
This isn't required anymore because the return type of the function is NSString.